index.html

```diff
@@ -1343,115 +1343,92 @@
 
-### Handling information that's being removed {#unacceptable-information}
+If existing APIs provide access to some information, but there is a plan to
+change those APIs to prevent that access, new APIs must not be added that
+provide that same information, unless it is possible to put additional
+[=appropriate=] [=access guards=] in place.
 
-<div class="practice">
-  <span class="practicelab">If existing APIs provide access to some information, but
-  we have a plan to change those APIs to prevent that access, new APIs must not be added
-  that provide that same information without extra [=access guards=] that make the access to the
-  information acceptable.</span>
-</div>
+For example, browsers are gradually removing the ability to join identities
+between different [=partitions=]. It is important that new APIs do not add
+features which re-enable [=cross-context recognition=].
 
-### Handling information we can't completely block {#sketchy-information}
+Some functionality of the web has historically been provided using features
+that can be used to undermine people's privacy (e.g. third-party cookies). It
+is not yet possible to remove access to all of the information that it would be
+better not to expose.
 
-<div class="practice">
-  <span class="practicelab">New APIs that provide access to undesirable information should
-  not make that information easier to access, unless they add [=access guards=] that make the
-  information acceptable.</span>
-</div>
+<aside class="example">
+  Some users are disappointed that the page they're visiting can discover which link
+  they clicked to leave that page. We can't block that information
+  because the page can use <a data-cite="RFC9110#status.3xx">HTTP redirects</a> to
+  learn it, and redirection is a core feature of the web.
+
+  Some users are disappointed that a page with permission to run JavaScript can record
+  their pattern of interaction with that page. However, the page does this by using
+  the same events it would use to make the page interactive, so we can't block this
+  information access either.
+</aside>
+
+New APIs that unavoidably provide access to this kind of information should
+not make that information easier to access, and nor should these APIs be
+easier to use by developers, compared to existing comparable web platform features.
+
+These APIs should also:
+
+* make it clear how to remove this access in the event that future web
+  platform changes make it possible to remove other access to the same
+  information.
+* make it clear how any [=user agent=] which blocks access to this kind of
+  information in general (perhaps by breaking some experiences on the web that
+  other browsers don't wish to break) can prevent the new API from exposing that
+  information without breaking additional sites or user experiences.
 
+<aside class="example">
+  Usually, these APIs will be designed to expose data that enables some
+  [=appropriate=] information discovery.
```

Comment:
Trying to make the web-without-3p-cookies reference flow better:

```suggestion
  Usually, these APIs will be designed to expose data that enables some
  [=appropriate=] information discovery, as recommended by [[[web-without-3p-cookies]]]:
  
  > It is better to approach [these use cases] with replacement technologies
  that are designed-for-purpose and built to respect user privacy.
```